feed.php

```diff
@@ -65,8 +65,21 @@ function ts_to_iso8601($ts)
 
 $feed_id= isset($_GET['feed']) ? $_GET['feed'] : '';
 $image_only=isset($_GET['image_only']);
+$image_random=isset($_GET['random']);
```

Comment:
$image_random=isset($_GET['random'])?true:false;

and the "if" below is unnecesary.